app/src/main/java/com/goldenraven/devkitwallet/data/Wallet.kt

```diff
@@ -123,5 +125,13 @@ object Wallet {
 
     fun getLastUnusedAddress(): AddressInfo = wallet.getAddress(AddressIndex.LAST_UNUSED)
 
-    fun isBlockChainCreated() = ::blockchain.isInitialized
+    fun isBlockChainCreated() = ::electrumServer.isInitialized
+
+    fun getElectrumURL(): String = electrumServer.getElectrumURL()
+
+    fun getElectrumSettings(): Boolean = electrumServer.getElectrumSettings()
+
+    fun setElectrumSettings(electrumSettings: Boolean) {
```

Comment:
As discussed during the dev call, let's name this method more explicitly, and use an enum instead of a boolean.